README.md

```diff
@@ -114,6 +114,43 @@ across the entire platform.
 
 Amazon XRAY also has a unique tracing ID that is propagated across the requests and can be tracked as well.
 
+## Customize request tracing
+
+You can customize the data that is tracked for each request by adding a global request mixin. 
+The request mixin takes the Lambda `event` and `context` and returns an object.
+This differs from the built in [pino mixin](https://github.com/pinojs/pino/blob/master/docs/api.md#mixin-function) as it only executes
+once per request where the built in pino mixin runs once per log entry.
+
+```ts
+import pino from 'pino-lambda';
+const logger = pino({
+  requestMixin: (event, context) => {
+    return {
+      // add request header host name
```

Comment:
Dumb question: is the return value always a map of headers, or can it be arbitrary data?

Reply:
I think that'd be nice! Maybe even adding a property in the example that doesn't look like/represent a header?